nipype/interfaces/fsl/model.py

```diff
@@ -1134,28 +1134,28 @@ class L2Model(BaseInterface):
 
     def _run_interface(self, runtime):
         cwd = os.getcwd()
-        mat_txt = ['/NumWaves       1',
-                   '/NumPoints      %d' % self.inputs.num_copes,
-                   '/PPheights      %e' % 1,
+        mat_txt = ['/NumWaves   1',
+                   '/NumPoints  {}'.format(self.inputs.num_copes),
+                   '/PPheights  1',
```

Comment:
Scientific notation is not according to the [FSL specifications](http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/GLM/CreatingDesignMatricesByHand). I don't know who made the decision to use a float there, especially because `/NumPoints` cannot be anything but an integer.